dev/benchmarks/complex_layout/ios/Runner/Info.plist

```diff
@@ -41,5 +41,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>io.flutter.embedded_views_preview</key>
+	<true/>
```

Comment:
This should be done in a different application.  This will impact other benchmarks besides this one, since it changes the threading model used by the application on iOS.

Reply:
This will become the default though. cc @cyanglaz what is the status of that?

Reply:
Still have several PRs waiting on different people to reviews. It might take some time as people have limited availability working from home. If you want to land this benchmark soon, we should create a different App.